samples/bootloader/src/main.c

```diff
@@ -16,8 +16,40 @@
 #include "fw_metadata.h"
 
 #include <fprotect.h>
-
 #include <provision.h>
+#include <image.h>
+
+#define MCUBOOT_VER_TO_U32(version) ((uint32_t\
```

Comment:
Where is this used?

Reply:
nowhere it's a leftover atm. The question is wouldn't it be more efficient to do a u32_t conversion and comparison than going through all the code paths of `mcuboot_is_version_sufficient` tbh I'm not sure.